geoalchemy2/elements.py

```diff
@@ -113,6 +113,35 @@ def desc(self):
         return binascii.hexlify(self.data)
 
 
+class EWKBElement(_SpatialElement, functions.Function):
+    """
+    Instances of this class wrap a EWKB value. Geometry values read
+    from the database are converted to instances of this type. You may
+    need to create instances of ``EWKBElement`` yourself.
+
+    Note: you cannot create ``EWKBElement`` objects directly from Shapely
```

Comment:
I now realize that the `WKBElement` does mention Shapely. So I guess you can ignore the above the comment.